arb_os/gasAccounting.mini

```diff
@@ -1251,21 +1174,21 @@ public func deferredPayments_new() -> DeferredPayments {
     return stack_new();
 }
 
-public func deferredPayments_add(dp: DeferredPayments, recipient: address, amount: uint) -> DeferredPayments {
-    return stack_push(dp, (recipient, amount));
+public func deferredPayments_add(dp: DeferredPayments, escrowKey: uint, recipient: address, amount: uint) -> DeferredPayments {
```

Comment:
Looks like this is only ever called with `EscrowStoreID_txGas` as the `escrowKey`. If that's the only way this will be used, might be better to hardcode that here rather than passing it in at all the callsites

Reply:
I'd prefer to keep this as is, so it's easier to re-use the deferred payments mechanism later for other purposes.